application/api/users.go

```diff
@@ -5,12 +5,9 @@ import (
 	"github.com/gofrs/uuid"
 )
 
-// swagger:model
-type Users []User
-
 // Attributes of a user of the App
```

Comment:
maybe add to the comment to differentiate from the "public" type. Maybe "Full list of user attributes, including private information." or something like that.

Reply:
I think the User is now the "public" type, whereas "UserPrivate" is "private", but the comments should indeed be more clear as to which role the User fulfills